droidmediacodec.cpp

```diff
@@ -508,6 +508,27 @@ class DroidMediaCodecBuilder {
     }
   }
 
+  static uint32_t flags(DroidMediaCodecMetaData *meta, uint32_t currentFlags) {
```

Comment:
Why is this moved?

Reply:
The function was private and I wanted to reuse it so I moved it and made it public and static.